app/mattermost.js

```diff
@@ -385,41 +385,55 @@ const handleSwitchToDefaultChannel = (teamId) => {
 };
 
 const launchSelectServer = () => {
-    Navigation.startSingleScreenApp({
-        screen: {
-            screen: 'SelectServer',
-            navigatorStyle: {
-                navBarHidden: true,
-                statusBarHidden: false,
-                statusBarHideWithNavBar: false,
-                screenBackgroundColor: 'transparent',
+    Navigation.setRoot({
+        root: {
+            stack: {
+                children: [{
+                    component: {
+                        name: 'SelectServer',
+                        passProps: {
+                            allowOtherServers: app.allowOtherServers,
+                        },
+                    },
+                }],
+                options: {
+                    layout: {
```

Comment:
what about the style options?

Reply:
That's a good question...

Reply:
I would like to have it in this PR as it will serve as a base for other PR's, in the meantime, can you base your work of this branch?

Reply:
Sorry, I read Elias's original comment wrong. These are the style options, but they've been moved around. The only one that isn't included is `statusBarHideWithNavBar` since I think it defaults to false. I don't think there's any other changes needed to this PR.